README

```diff
@@ -60,7 +60,7 @@ $ syt pull /path/to/remote/repo
 Testing
 
 Run the tests:
-$ python3 -m unittest discover syncthing '*_spec.py'
+$ python3 -m unittest discover syncthing -t . -p '*_spec.py'
```

Comment:
-p == --pattern, -t == --top-level-directory; the suggestion might leave syncthing out... but then someone will probably try to add syncthing again and miss the -t . option, so i left it in there.  Since you always need the --pattern for our non-default *_spec* files, one might better put it first.

```suggestion
$ python3 -m unittest discover --pattern '*_spec.py' --top-level-directory .
```
...which also demonstrates that feature.